app/controllers/application.js

```diff
@@ -120,7 +120,22 @@ export default Ember.Controller.extend(mapQueryParams.Mixin, {
 
       if (feature) {
         if (highlightedLayer === feature.layer.id) {
-          const { bbl, ulurpno, zonedist, sdlbl, splbl, overlay, cartodb_id } = feature.properties;
+          const {
+            bbl,
+            ulurpno,
+            zonedist,
+            sdlbl,
+            splbl,
+            overlay,
+            cartodb_id,
+            ceqr_num,
+          } = feature.properties;
+
+          //  handle e-designation click
+          if (ceqr_num) { // eslint-disable-line
+            window.open(`http://a030-lucats.nyc.gov/lucats/DirectAccess.aspx?CEQRNO=${ceqr_num}`) // eslint-disable-line
```

Comment:
@chriswhong you should add a little defensive programming with the `window` object in case it does not exist for node-based environments for whatever reason `window !== 'undefined' && window.open`. 

In terms of UX, does it make sense to send users off-site after clicking on the map? Thoughts?

Reply:
It's either that or we add a route for e-designations.  I don't think there's anything wrong with linking out directly from a map click as long as it opens a new window/tab

Reply:
Okay - our app doesn't do this anywhere else and external links are always clearly marked. I think it's awkward, but it's your call.

Reply:
I killed it

Reply:
Sorrrrryyyy